File: second-year/second-semester/MPP/mpp-proiect-repository-raulbaciulescu/travelAgency2/model/Location.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace model
{
    [Serializable]
    public class Location : Entity<long>
    {
        public string name { get; set; }
        public string airport { get; set; }

        public Location(long id, string name, string airport) : base(id)
        {
            this.name = name;
            this.airport = airport;
        }
        public Location() : base(0L)
        {
            
        }
        
        public Location(string name) : base(0L)
        {
            this.name = name;
            airport = "";
        }

        public override string ToString()
        {
            //base.ToString() + ": " +
            return name + " " + airport;
        }
    }
}

File: second-year/second-semester/MPP/mpp-proiect-repository-raulbaciulescu/travelAgency/Model/src/main/java/domain/Location.java
package domain;

import java.io.Serializable;

public class Location extends Entity<Long> implements Serializable {
    private String name;
    private String airport;

    public Location(Long id, String name, String airport) {
        this.setId(id);
        this.name = name;
        this.airport = airport;
    }

    public Location() {

    }

    public Location(String name, String airport) {
        this.name = name;
        this.airport = airport;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getAirport() {
        return airport;
    }

    public void setAirport(String airport) {
        this.airport = airport;
    }

    @Override
    public String toString() {
        return name + " " + airport;
    }
}
